server/config.js

```diff
@@ -6,7 +6,8 @@ const cache = new Map()
 const defaultConfig = {
   webpack: null,
   poweredByHeader: true,
-  distDir: '.next'
+  distDir: '.next',
+  assetPrefix: ''
```

Comment:
it might be a function because we need to add the .json as well as rewriting / to /index, or does that happen automatically?